compiler/plugins/target/AMD-AIE/iree-amd-aie/Target/XCLBinGen.cpp

```diff
@@ -467,18 +527,18 @@ static LogicalResult generateCoreElfFiles(
         std::string targetFlag = "--target=" + targetLower + "-none-elf";
         flags.push_back(targetFlag);
 
-        if (doUseMeBasic) {
-          SmallString<64> meBasicPath(peanoDir);
-          sys::path::append(meBasicPath, "lib",
-                            targetLower + "-none-unknown-elf", "me_basic.o");
-          flags.emplace_back(meBasicPath);
+        if (buildCRT(peanoDir, verbose)) {
+          auto crtObjFile = assembleStringUsingPeano(
```

Comment:
I dont follow how this will work with old peano that uses me_basic.o for start up code, is it equivalent to crt.o now?

Reply:
I see you had the answer to this in description :)